integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -263,15 +249,7 @@ object CarbonDataRDDFactory {
             throw new Exception("Exception in compaction " + exception.getMessage)
           }
         } finally {
-          executor.shutdownNow()
-          try {
-            compactor.deletePartialLoadsInCompaction()
```

Comment:
other than removing the `deletePartialLoadsInCompaction`, rest of the changes are nothing to do with clean files PR. can you raise a separate PR for this?

Reply:
better to add proper description in the PR and handle here only, instead of handling again in other PR, as review will be easy and to avoid duplicate working, should be fine @ajantha-bhat ?

Reply:
ok, I see that many changes are done because to support this `DataTrashManager.cleanStaleCompactionSegment`

Reply:
already add code to handle compaction exception instead of this function.
this function will list the whole table to clean partial loads.
new function only focus on current load.